client/app/screens/FaqScreen.js

```diff
@@ -2,8 +2,9 @@ import React, { Component } from 'react';
 import BottomNavigation, { Tab, NavigationComponent } from 'react-native-material-bottom-navigation'
 import { TabNavigator, StackNavigator } from 'react-navigation'
 
-import { Header } from '../components/header'
-import { FaqList, FaqDetails } from '../components/faq'
+import Header from '../components/Header'
+import FaqList from '../components/faq/FaqList'
```

Comment:
Same issue with capitalization of FaqList and FaqDetails

Reply:
Oops, for some reason, I forgot to capitalize these files.  Thanks Amy!